### PR TITLE
Create vtx.json

### PR DESCRIPTION
**Pull request by lcrownover**

### CODE REVIEW

File: include/vtx.json [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+	"bands": "FEAB"
+}
+
+}
+	"id": 7,
+	"brand": "Team BlackSheep",
+	"model": "Unify 5G8R",
+	"wattage": "200",
+	"channels": "32",
+	"bands": "FEAR"
```
Comment (natsn) [inline include/vtx.json]: FEAR?

**Comment by lcrownover** [comment]
F - Fatshark
E - Boscam_e/DJI
A - Boscam_a
B - Boscam_b
R - Raceband

**Comment by natsn** [comment]
I like it! That makes things easier. And great Idea putting all the data in as json

**Comment by natsn** [comment]
So far we are splitting the profits 50/50 ;)

**Comment by lcrownover** [comment]
haha, that's the goal :)

**Comment by lcrownover** [comment]
do you see a benefit to keeping the ID field? it's a pain to edit and I
don't see any reason we need it.

**Comment by natsn** [comment]
No need to have the ID field. 

This is a great plugin though for dealing with the pain you describe: https://github.com/yulanggong/IncrementSelection

**Comment by natsn** [comment]
btw - should be able to create feature branches and push to this repo now - i added you as a collaborator.

**Comment by lcrownover** [comment]
coolio

**Comment by lcrownover** [comment]
I'm assuming you've heard of framework7?

http://www.idangero.us/framework7/#.VjQnjIpHanM

**Pull request merged**